src/docfx/lib/git/Repository.cs

```diff
@@ -19,7 +19,7 @@ private Repository(string remote, string? branch, string commit, PathString path
         {
             // remove user name, token and .git from url like https://xxxxx@dev.azure.com/xxxx.git
             Remote = Regex.Replace(remote, @"^((http|https):\/\/)?([^\/\s]+@)?([\S]+?)(\.git)?$", "$1$4");
-            Branch = branch ?? "master";
```

Comment:
Is it possible to simply change to `main`?

Reply:
We can, but it is not necessary, we prefer to keep the real value if we can.